wordpress-mobile/WordPress-Android
# issue/6694 Sharing does not continue after login

Fixes #6694 - Add support for login while sharing to WordPress

To test:
With the app data cleared, try to share a photo to WordPress from the device's gallery. The app should let you log in and continue with the sharing action.
